v0.3/handbook/libraries/stdlib.md

```diff
@@ -1456,4 +1453,180 @@ bind
   </tr>
 </table>
 
+## File
+
+Watchers that access the filesystem. These watchers only work in headless mode.
+
+<table class="libitem">
+  <tr>
+    <td colspan="2">
+      <b>#file/read</b> - read the specified file
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>path</strong> - The path of the file to be read.</li>
+        <li><strong>encoding</strong> - (optional) - The encoding of the file. The default is utf-8.</li>
+        <li><strong>error</strong> - (optional) - If an error is encountered when attempting to read the file, it will be stored here as a #file/error.</li>
+        <li><strong>contents</strong> - The contents of the file. This attribute will have a value once the entire contents of the file are read.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Read a file
```

Comment:
I've got a branch that takes these out, and formats the examples as standard Eve. Putting the // isn't necessary outside a block